catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -99,6 +99,8 @@
      */
     public static final AttributeType<Short> SHORT_TYPE;
 
+    public static final AttributeType<HashMap> HASHMAP_TYPE;
```

Comment:
Since I shouldn't be using a `HashMap` and `Map` isn't serializable, would you guys think it a better idea to instead keep track of a list of a `Serializable` object that contains the validator name, boolean for errors, and boolean for warnings?